solutions/usaco-760.mdx

```diff
@@ -117,4 +118,35 @@ int main(){
 ```
 
 </CPPSection>
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
```

Comment:
Move this to where the existing `## Implementation` is, under official analysis.